cr.h

```diff
@@ -370,11 +369,53 @@ struct cr_plugin {
 
 #include <cassert>                 // assert
 #include <chrono>                  // duration for sleep
-#include <experimental/filesystem> // fs::path and utils
-#include <system_error>            // filesystem errors
 #include <thread>                  // this_thread::sleep_for
+#include <string>
+#include <algorithm>
 
-namespace fs = std::experimental::filesystem;
+#if _WIN32
+#define CR_SEPARATOR '\\'
+#else
+#define CR_SEPARATOR '/'
+#endif
+
+static void cr_split_path(std::string path, std::string &parent_dir,
+                          std::string &base_name, std::string &ext) {
+#if _WIN32
+    std::replace(path.begin(), path.end(), '/', CR_SEPARATOR);
```

Comment:
You can use another define for the character being replaced with the one in 376-380 and avoid this ifdef here, something like: CR_INVALID_SEPARATOR or CR_FROM_SEPARATOR.